packages/sui-tealium-tracker/test/SuiTealiumTrackerSpec.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-env mocha */
+import {expect} from 'chai'
+import sinon from 'sinon'
+import {SuiTealiumTracker} from '../src/SuiTealiumTracker'
+
+describe.only('SuiTealiumTracker', () => {
+  let suiTealiumTracker
+  let sendTealiumThrottledSpy
+  before(() => {
+    const fixture = `
+<div id='fixture'>
+  <button id='tealium-click-bugged' data-tealium-bugged-tag='test_tealium_tagging'>Simple Div Tagging</button>
+  <button id='tealium-click' data-tealium-tag='test_tealium_tagging'>Simple Div Tagging</button>
+</div>
+  `
+    document.body.insertAdjacentHTML('afterbegin', fixture)
+    suiTealiumTracker = new SuiTealiumTracker('test')
+    suiTealiumTracker.init()
+    sendTealiumThrottledSpy = sinon.spy(suiTealiumTracker, 'sendTealiumThrottled')
+  })
+
+  it('should set our customEvent var to test', () => {
+    expect(suiTealiumTracker.customEventName).to.equal('test')
+  })
+
+  it('should call tealium if our button have the tag placed', (done) => {
+    document.getElementById('tealium-click').click()
+    setTimeout(() => {
```

Comment:
please avoid setTimeout in unit test, use fake timers as you are already loading sinon